feature-utils/silly-i18n/rollup-plugin.cjs

```diff
@@ -0,0 +1,57 @@
+const fs = require("fs");
+const path = require("path");
+const util = require("util");
+const glob = require("glob");
+
+const DIRNAME = "locales";
+const FILEEXT = ".json";
+const PLUGIN = "silly-i18n";
+const SUFFIX = `?${PLUGIN}`;
+
+exports.default = function (options) {
+    return {
+        name: PLUGIN,
+
+        async resolveId(source, importer) {
+            if (source == `!${PLUGIN}`) {
```

Comment:
OK, so this function is running over *every* file out there (can be quite a few) looking for a pattern (not documented elsewhere), and doing so *every* time a build is made, even if the files have not changed (they tend not to change a lot). Is that correct?

Reply:
Yes, this is generally how a Rollup plugin that handles module imports works.

Reply:
Again, it's not a prerequisite that we create a rollup plugin. We only do so if it's an option with less tradeoffs that other possible options.

Reply:
I don't get what you are suggesting then? Either we generate the i18n configuration and Rollup provides a straight-forward framework to do so, or we leave everything as-is in favor of not having to generate code to be bundled, and close this ticket without a resolution. It seems you are in favor of the latter?